enterprise/server/auth/auth.go

```diff
@@ -472,19 +481,46 @@ func lookupUserFromSubID(env environment.Env, ctx context.Context, subID string)
 		if err := userRow.Take(user).Error; err != nil {
 			return err
 		}
-		groupRows, err := tx.Raw(`SELECT g.* FROM `+"`Groups`"+` as g JOIN UserGroups as ug
-                                          ON g.group_id = ug.group_group_id
-                                          WHERE ug.user_user_id = ?`, user.UserID).Rows()
+		groupRows, err := tx.Raw(`
+			SELECT
+			  g.user_id,
+				g.group_id,
+				g.url_identifier,
+				g.name,
+				g.owned_domain,
+				g.github_token,
+				g.sharing_enabled,
+				g.use_group_owned_executors,
+				g.saml_idp_metadata_url,
+				ug.role
```

Comment:
FYI this is selecting all the fields from Group except

* Model fields (created_at_usec, updated_at_usec) -- we don't reference these anywhere
* WriteToken and APIKey -- these aren't used anywhere either; they can be removed in a separate PR